tests/test_condor_unit.py

```diff
@@ -115,25 +115,29 @@ class TestCondorUnit:
 
     # lib/condor.py routines...
 
+    @pytest.mark.unit
     def test_get_schedd_1(self):
         """make sure we get our test schedd back with test_vargs"""
         schedd = condor.get_schedd(TestUnit.test_vargs)
         print("got schedd: {0}".format(schedd))
         print("schedd name: {0}".format(schedd["Name"]))
         assert schedd["Name"] == TestUnit.test_schedd
 
+    @pytest.mark.unit
     def test_load_submit_file_1(self, get_submit_file):
         """make sure load_submit_file result has bits of the submit file"""
         res = condor.load_submit_file(get_submit_file)
         assert str(res[0]).find("universe = vanilla") >= 0
         assert str(res[0]).find("executable = /bin/true") >= 0
 
+    @pytest.mark.unit
```

Comment:
And I'd consider this an integration test, since it requires a working condor cluster.

Reply:
Well, it only tests one unit of the code; but the test does have an external dependency. 
This is also true of several of the file-upload tests, we actually upload a tarfile...
Building mocks for those webservices and giving ourselves a way to use those mocks would
let us test them standalone, and also speed up tests, since we wouldn't have to actually wait for
cvmfs servers, etc.

Reply:
But I think that's a separate issue/pull request ;-)